solutions.html

```diff
@@ -0,0 +1,147 @@
+                <div class="carousel__title services__title inria">{{ service.title }}</div>
+                <div class="carousel__description">{{ service.description }}</div>
+                <div class="carousel__button">
+                  <a class="button" href="/">{{site.data.solutions.industries.more_button}}</a>
+                </div>
+              </label>
+            {% endfor %}
+          </div>
+        </div>
+        <p class="title-3">{{site.data.solutions.industries.subtitle_2}}</p>
+        <h2 class="title-2">{{site.data.solutions.project_development.title}}</h2>
+        <p class="title-3">{{site.data.solutions.project_development.description}}</p>
+      </div>
+    </div>
+    <div class="solutions__project-development">
+      <div class="wrapper timeline">
+        {% for item in site.data.solutions.project_development.approaches %}
+        <div class="timeline-item">
+          <div class="timeline-item__icon">
+            <img src="assets/images/solutions/{{ item.image }}.webp" alt="Approach">
+          </div>
+          <div class="timeline-item__content">
+            <div class="timeline-item__content-title inria">{{ item.title}}</div>
+            <div class="timeline-item__content-description">{{ item.description}}</div>
+          </div>
+          <div class="timeline__processes">
+            {% for process in item.processes%}
+              <div class="timeline__processes-item">
+                {{process.item}}
+                <div class="arrow">→</div>
+              </div>
+            {% endfor %}
+          </div>
+        </div>
+      {% endfor %}
+      </div>
+    </div>
+    <div class="solutions__staff-augmentation">
+      <div class="row wrapper">
+        <h3 class="title-2">{{ site.data.solutions.staff_augmentation.title }}</h3>
+        <p class="base-content">{{ site.data.solutions.staff_augmentation.description }}</p>
+        <div class="timeline wrapper">
+          {% for item in site.data.solutions.staff_augmentation.approaches %}
+            <div class="timeline-item">
+              <div class="timeline-item__icon">
+                <img src="assets/images/solutions/{{ item.image }}.webp" alt="Approach">
+              </div>
+              <div class="timeline-item__content">
+                <div class="timeline-item__content-title inria">{{ item.title}}</div>
+                <div class="timeline-item__content-description">{{ item.description}}</div>
+              </div>
+              <div class="timeline__processes">
+                {% for process in item.processes%}
+                  <div class="timeline__processes-item">
+                    {{process.item}}
+                    <div class="arrow">→</div>
+                  </div>
+                {% endfor %}
+              </div>
+            </div>
+          {% endfor %}
+        </div>
+      </div>
+      <a class="cto-button" href="/services.html">{{ site.data.solutions.staff_augmentation.more_button }}</a>
```

Comment:
footer is moving when hover on the button
![image](https://github.com/katatsu12/live_blog/assets/22080708/7a0c9fc0-a9b4-4fa5-a532-9698195dd241)

Reply:
Ok